jgt_tools/run_tests.py

```diff
@@ -16,14 +16,15 @@
 __doc__ = __doc__.format("\n    ".join(__commands_to_run))
 
 
-def run_tests(do_setup=False, self_check=False, verbose=True):
+def run_tests(do_setup=False, self_check=False, verbose=True, additional_args=""):
```

Comment:
Ha! Good catch. I think initially I was thinking it would be the full string, it ended up being a list, and I just forgot to make the necessary change here.